test/functions/ferguson/parenless-call-bug.chpl

```diff
@@ -0,0 +1,33 @@
+// Bug reproducer from issue #25266
+// This code is based upon the test
+//  parallel/forall/may-vs-must-par/promo-use-standalone.chpl
+
+config const ni = 2;
+const RNGi = 1..ni;
+
+proc inc1i { }
```

Comment:
Can we make the function print or something so that we know it's actually called instead of elided due to dead code elimination or something?  As long as that still triggered the bug before, of course